samples/packages/awscdk-typescript/src/construct/services.ts

```diff
@@ -22,7 +22,10 @@ export interface ServicesProps {
 }
 
 export class Services extends Construct {
-  readonly cluster: Cluster;
+  private _cluster: Cluster;
+  public get cluster(): Cluster {
+    return this._cluster;
+  }
```

Comment:
Interesting, the previous definition of a `readonly` field should work?

Reply:
Setting from inside an `if` block wasn't. Or maybe just local linter (?).

Reply:
Raising this as it becomes inconsistent with how other fields are defined. It should stay the same unless absolutely necessary I think.

Reply:
OK, it was the `find()` rather than the `if` it didn't work within, which makes sense. changed.